disputils/multiple_choice.py

```diff
@@ -68,6 +71,10 @@ def _generate_embed(self) -> discord.Embed:
             description=self.description,
             color=self.color
         )
+        if self.footer_text:
+            config_embed.set_footer(text=self.footer_text, icon_url=self.footer_icon)
+        if self.embed_timestamp:
+            config_embed.timestamp = self.embed_timestamp
```

Comment:
Please explain to me, why you would want to set the timestamp! How would this effect the resulting embed? I never did this and I never saw anyone doing it.

Reply:
It is mainly because I use a bot that has timestamps and footers on all embeds. I noticed that you can't add timestamps to the multi choice embed so I thought it would be a good idea to add that functionality.

Reply:
Oh well, I didn't know what actually happens when you add a timestamp to the embed, but now I tried it myself and I'm surprised I didn't even know that was possible. :sweat_smile: 

Anyway, now that I know what it does, it is indeed a pretty good idea to make it work with the multiple choice embed.